src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -929,9 +930,9 @@ public static FormulaValue Split(IRContext irContext, StringValue[] args)
         }
 
         // This is static analysis before actually executing, so just use string lengths and avoid contents. 
-        internal static int SubstituteGetResultLength(int sourceLen, int matchLen, int replacementLen, bool replaceAll)
+        internal static double SubstituteGetResultLength(int sourceLen, int matchLen, int replacementLen, bool replaceAll)

```

Comment:
>SubstituteGetResultLength [](http://example.com/codeflow?start=31&length=25)

For overflow,  should this be a checked context? 


We should still catch the overflow exception

[https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/checked-and-unchecked](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/checked-and-unchecked)

Reply:
add check overflow.